packages/data/src/types/abstract-query.ts

```diff
@@ -34,9 +34,3 @@ export interface AbstractQuery {
  * @TODO
  * - Rethink every / some
  */
```

Comment:
@jaads Probably unrelated to this PR, but this comment feels also out of place

(https://github.com/directus/directus/pull/19941/files#diff-30f1d3b46baaef5d5605d82b0c67c65853cbd90f6092ee92563ca1b067359e35R10-R36)